ratis-examples/src/main/java/org/apache/ratis/examples/counter/server/CounterServer.java

```diff
@@ -52,6 +52,7 @@ public CounterServer(RaftPeer peer, File storageDir) throws IOException {
 
     //set the storage directory (different for each peer) in the RaftProperty object
     RaftServerConfigKeys.setStorageDir(properties, Collections.singletonList(storageDir));
+    RaftServerConfigKeys.Read.setOption(properties, RaftServerConfigKeys.Read.Option.LINEARIZABLE);
```

Comment:
Please add a comment to explain what this parameter does

Reply:
Done